app/views/brexit_checker/_results_criteria.html.erb

```diff
@@ -1,22 +1,21 @@
 <% if criteria.present? %>
-  <aside
-    class="govuk-grid-column-one-third"
-    aria-label=<%="#{criteria_aside_label(local_assigns[:heading])}"%>
-  >
-    <% if local_assigns[:heading] %>
-      <h3 class="govuk-heading-m"><%= heading %></h3>
-    <% end %>
-    <p class="govuk-body brexit-checker__critera_explaination">
-      <%= t('brexit_checker.results.criteria.list_context') %>
-    </p>
-    <ul class="govuk-list govuk-list--bullet">
-      <% criteria.each do |criterion| %>
-        <li class="brexit-checker__criterion">
-          <%= criterion.text %>
-        </li>
-      <% end %>
-    </ul>
-  </aside>
+  <div class="govuk-grid-column-one-third">
+  <% if local_assigns[:heading] %>
+    <h3 class="govuk-heading-m"><%= heading %></h3>
+  <% end %>
+    <aside>
```

Comment:
If this is an `aside`, shouldn't it include the heading?

Reply:
I think the heading belongs outside the aside, because it refers to the actions in the section as well as the aside bit:

<img width="1038" alt="Screenshot 2020-09-24 at 15 35 30" src="https://user-images.githubusercontent.com/773037/94159647-b2a7df00-fe7b-11ea-8ed8-4af2fd033ba2.png">

So "Visiting Ireland" is the h3, the "Because you said" bit is the aside, and the "Check whether your mobile phone" bit is subordinate to the h3.

So "Visiting Ireland" is key to the page and the aside is assistive. I thought my way would be more appropriate. Though I may be wrong :-D

Reply:
That provides a bit more context, thanks. Genuinely I don't know, now.

Reply:
I think moving the heading out of the aside makes sense :+1:

Q: Is it fine that we are removing the aria label, as opposed to amending it?

Reply:
I don't know what it was supposed to add, really?